install.sh

```diff
@@ -260,6 +273,38 @@ set_synchrony_url() {
   fi
 }
 
+# Update the current load balancer listener on port 7999 to use the TCP protocol
+enable_tcp_protocol_on_lb_listener() {
+  local install_bitbucket
+  local region
+  local load_balancer_dns
+  local load_balancer_name
+  local original_instance_port
+
+  install_bitbucket=$(get_product "bitbucket" "${CONFIG_ABS_PATH}")
+
+  if [ -n "${install_bitbucket}" ]; then
+    region=$(get_variable 'region' "${CONFIG_ABS_PATH}")
+    load_balancer_dns=$(terraform output | grep '"load_balancer_hostname" =' | sed -nE 's/^.*"(.*)".*$/\1/p')
+    load_balancer_name=$(echo "${load_balancer_dns}" | cut -d '-' -f 1)
+    original_instance_port=$(aws elb describe-load-balancers --load-balancer-name "${load_balancer_name}" --query 'LoadBalancerDescriptions[*].ListenerDescriptions[*].Listener[]' --region "${region}" | jq '.[] | select(.LoadBalancerPort==7999) | .InstancePort')
```

Comment:
Today I have learned that the query language in `aws cli` is actually quite powerful https://jmespath.org/ I haven't used it much but with a bit of testing I was able to get to:

```suggestion
    original_instance_port=$(aws elb describe-load-balancers --load-balancer-name "${load_balancer_name}" --query 'LoadBalancerDescriptions[*].ListenerDescriptions[?Listener.LoadBalancerPort==`7999`].Listener[].InstancePort | [0]' --region "${region}"
```

This removes the dependency on `jq` completely and we can remove it from the documentation. I have tested this locally, but please verify it solve it with the changes as well.

Reply:
So is jmspath built into the `aws cli`?

Reply:
It is https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-filter.html